### PR TITLE
 logging-enabled container group (wip)

### PR DESCRIPTION
**Pull request by TruaShamu**

### CODE REVIEW

File: InferenceSystem/deploy-aci.yaml [hunk 1]
```diff
@@ -4,61 +4,78 @@ identity: null
...
-        requests:
-          cpu: 1.0
-          memoryInGB: 3
-  imageRegistryCredentials:
-  - server: orcaconservancycr.azurecr.io
-    username: orcaconservancycr
-    password: '<image_registry_password>'
-  osType: Linux
-  restartPolicy: Always
+	containers:
```
Comment (micya) [inline InferenceSystem/deploy-aci.yaml:7]: Can you change from tabs to whitespace for consistency?

File: InferenceSystem/deploy-aci.yaml [hunk 2]
```diff
@@ -4,61 +4,78 @@ identity: null
...
+			ports: []
+			resources:
+				requests:
+					cpu: 1.0
+					memoryInGB: 1.5
+	osType: Linux
+	restartPolicy: Always
+	diagnostics:
+		logAnalytics:
+			workspaceId: 8a090d5a-065e-4659-bc0b-7e6ecc7597a2
```
Comment (micya) [inline InferenceSystem/deploy-aci.yaml:66]: Please don't put ID and key as part of repo. These are considered secrets and should only have placeholders in commit.

File: InferenceSystem/deploy-aci.yaml [hunk 3]
```diff
@@ -4,61 +4,78 @@ identity: null
...
+				secureValue: '<storage_connection_string>'
+			- name: INFERENCESYSTEM_APPINSIGHTS_CONNECTION_STRING
+				secureValue: '<appinsights_connection_string>'
+			image: orcaconservancycr.azurecr.io/live-inference-system:11-15-20.FastAI.R1-12.BushPoint.v0
+			ports: []
+			resources:
+				requests:
+					cpu: 1.0
+					memoryInGB: 3
+	- name: inference-logs
```
Comment (micya) [inline InferenceSystem/deploy-aci.yaml:53]: I don't believe creating an additional container in the group is necessary. This is a sample application for the example.

File: InferenceSystem/deploy-aci.yaml [hunk 4]
```diff
@@ -4,61 +4,78 @@ identity: null
...
+		properties:
+			environmentVariables: []
+			image: fluent/fluentd
+			ports: []
+			resources:
+				requests:
+					cpu: 1.0
+					memoryInGB: 1.5
+	osType: Linux
```
Comment (micya) [inline InferenceSystem/deploy-aci.yaml:62]: Some of these configurations are already specified elsewhere in the file. Please don't duplicate them.

**Review by micya — CHANGES_REQUESTED** [review]
Please also add some documentation in relevant README describing what the purpose of workspace is.

File: InferenceSystem/deploy-aci.yaml (hunk 1 above)
Comment (TruaShamu) [reply InferenceSystem/deploy-aci.yaml]: done

**Pull request closed without merging**